tools/collect_smileys.php

```diff
@@ -67,7 +67,6 @@ function collect_smileys($file) {
 	table.smiley_table td { font: 10pt Courier,monospace,mono; white-space: pre; }
 	table.smiley_table tbody th { text-align: center; }	
 --></style>
-<base href="<?php print "http://" . $_SERVER['SERVER_NAME'] . dirname(dirname($_SERVER['REQUEST_URI'])); ?>" />
```

Comment:
This line should not be removed. 

See https://github.com/vanilla/nbbc/pull/11/files#diff-028043b7bd7f006af37bbd7f9649a099R29